scipy/stats/_multicomp.py

```diff
@@ -24,10 +27,167 @@
 class DunnettResult:
     statistic: np.ndarray
     pvalue: np.ndarray
+    _alternative: Literal['two-sided', 'less', 'greater']
+    _rho: np.ndarray
+    _df: int
+    _std: float
+    _mean_samples: np.ndarray
+    _mean_control: np.ndarray
+    _n_samples: np.ndarray
+    _n_control: int
+    _rng: SeedType
+    _ci: ConfidenceInterval | None = None
+    _ci_cl: DecimalNumber | None = None
+
+    def __str__(self):
+        # Note: `__str__` prints the confidence intervals from the most
+        # recent call to `confidence_interval`. If it has not been called,
+        # it will be called with the default CL of .95.
```

Comment:
This follows the printy printing of `tukey_hsd` closely, right? I reviewed that carefully, so I think this is fine.